src/ralph/ssl_certificates/management/commands/update_service_env.py

```diff
@@ -0,0 +1,68 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from ralph.assets.models.assets import ServiceEnvironment
+from ralph.dns.dnsaas import dnsaas_client
+from ralph.ssl_certificates.models import SSLCertificate
+
+
+def checking_type(value):
+    if value['type'] is 'CAME' or 'PTR':
```

Comment:
it's not working as you intended - use `if value['type'] in {'CAME', 'PTR'}`

Reply:
and you probably mean `CNAME`

Reply:
and are you sure you want to analyze PTR records? Aren't `A`, `AAAA` and `CNAME` enough?

Reply:
I've changed it. We want to analyze PTR records. From time to time we're generating cert for this typo of record.